### PR TITLE
Make path relative after BibTeX-from-PDF import

### PR DESCRIPTION
**Pull request by koppor**
- Closes https://github.com/JabRef/jabref/issues/11173
- Closes https://github.com/JabRef/jabref/pull/11409

### Mandatory checks

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/importer/WebFetchers.java [hunk 1]
```diff
@@ -178,9 +175,13 @@ public static SortedSet<EntryBasedFetcher> getEntryBasedFetchers(ImporterPrefere
         set.add(new MathSciNet(importFormatPreferences));
         set.add(new CrossRef());
         set.add(new ZbMATH(importFormatPreferences));
-        set.add(new PdfMergeMetadataImporter.EntryBasedFetcherWrapper(importFormatPreferences, filePreferences, databaseContext));
         set.add(new SemanticScholar(importerPreferences));
         set.add(new ResearchGate(importFormatPreferences));
+
+        // Uses the PDFs - adn then uses the parsed Dpi. Makes it 10% a web fetcher.
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/logic/importer/WebFetchers.java:181]: Typos

File: src/main/java/org/jabref/logic/importer/fileformat/BiblioscapeImporter.java [hunk 2]
```diff
@@ -38,8 +38,7 @@ public StandardFileType getFileType() {
 
     @Override
     public String getDescription() {
-        return "Imports a Biblioscape Tag File.\n" +
-                "Several Biblioscape field types are ignored. Others are only included in the BibTeX field \"comment\".";
+        return "Imports a Biblioscape Tag File.\nSeveral Biblioscape field types are ignored. Others are only included in the BibTeX field \"comment\".";
```
[the remark this answers is not in the document]
Comment (koppor) [reply src/main/java/org/jabref/logic/importer/fileformat/BiblioscapeImporter.java]: "Some" localizations added: efd80e0fe0

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,14 +35,15 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
...
+- We fixed an issue where JabRef put file paths as absolute path when an entry was created using drag and drop of a PDF file. [#11173](https://github.com/JabRef/jabref/issues/11173)
 - We fixed an issue that online and offline mode for new library creation were handled incorrectly. [#11565](https://github.com/JabRef/jabref/pull/11565)
 - We fixed an issue with colors in the search bar when dark theme is enabled. [#11569](https://github.com/JabRef/jabref/issues/11569)
 - We fixed an issue where a new unsaved library was not marked with an asterisk. [#11519](https://github.com/JabRef/jabref/pull/11519)
 - We fixed an issue where JabRef starts without window decorations. [#11440](https://github.com/JabRef/jabref/pull/11440)
 
 ### Removed
 
-
+- We removed support for importing using the SilverPlatterImporter (`Record INSPEC`). [#11576](https://github.com/JabRef/jabref/pull/11576)
```
Comment (Siedlerchr) [inline CHANGELOG.md:46]: This should not be part of this PR and second, why? https://newsbreaks.infotoday.com/NewsBreaks/Wolters-Kluwer-Adds-SilverPlatter-Information-to-Its-Collection-17613.asp

**Review by Siedlerchr — CHANGES_REQUESTED** [review]

File: src/main/java/org/jabref/logic/util/io/FileUtil.java [hunk 4]
```diff
@@ -268,6 +270,30 @@ public static Path relativize(Path path, BibDatabaseContext databaseContext, Fil
         return relativize(path, fileDirectories);
     }
 
+    /**
+     * Relativizes all BibEntries given to (!) the given database context
+     * <p>
+     * ⚠ Modifies the entries in the list ⚠
+     */
+    public static List<BibEntry> relativize(List<BibEntry> entries, BibDatabaseContext databaseContext, FilePreferences filePreferences) {
```
[the remark this answers is not in the document]
Comment (koppor) [reply src/main/java/org/jabref/logic/util/io/FileUtil.java:278]: Added in 46c9197842

File: CHANGELOG.md (hunk 3 above)
Comment (koppor) [reply CHANGELOG.md:46]: Did not want the translators to translate something 🤣🤣

Historical note: Tests were added at https://github.com/JabRef/jabref/pull/510 - and

**Review by Siedlerchr — APPROVED** [review]

**Review by Siedlerchr — APPROVED** [review]
